src/credentials/GroupDataProviderImpl.h

```diff
@@ -80,6 +80,8 @@ class GroupDataProviderImpl : public GroupDataProvider
     // Key Sets
     //
 
+    static CHIP_ERROR DeriveOperationalCredentials(const ByteSpan & epoch_key, const ByteSpan & compressed_fabric_id,
```

Comment:
This function is misplaced, I believe it should go into CHIPCryptoPAL.h

Reply:
Moved as suggested, which forced moving `GroupOperationalCredentials` structure to `chip::Crypto` as well.